packages/main/src/models/conversation.ts

```diff
@@ -35,28 +35,33 @@ export function getReplyParticipants(
 ): Participants {
   const senderAddress = Addr.build(sender)!
   const participants = getParticipants(conversation)
-  const to = uniqBy(
-    Addr.normalizedEmail,
-    List(
-      participants
-        .replyTo!.concat(participants.to)
-        .filter(p => !Addr.equals(senderAddress, p))
-    )
-  )
-    .sortBy(Addr.formatAddress)
-    .toArray()
+  const to = participants.replyTo
```

Comment:
This is good thinking, but what if `participants.to` is defined? In that case we want to use those values for the `to` list for the new message.

I would move the ternaries to more specific sites like this:

```ts
uniqBy(
        Addr.normalizedEmail,
        List(
          (participants.replyTo || [])
            .concat(participants.to || [])
            .filter(p => !Addr.equals(senderAddress, p))
        )
      )
```